external-docs/docs/guides/errors/build-command-errors/attribute-not-linked.md

```diff
@@ -0,0 +1,11 @@
+# Error - Attribute Not Linked
+
+## When it occurs
+
+A units or attribute column is defined but it is not linked to any observation value column.
```

Comment:
Could you provide examples of the different kind of errors, for example could you write tables and JSON snippets to represent an example input to the csvcubed build command that would generate the error? Could you do this for each of the errors defined?

Try to be as tame as you can with the termms us